taskcluster/ci/push-apk/kind.yml

```diff
@@ -25,9 +26,13 @@ job-template:
     description: Publish Fenix
     worker-type: push-apk
     worker:
-        commit: true
+        commit:
+            by-build-type:
+                fennec-nightly: false
```

Comment:
Note: Push-apk will run but not change anything yet. This is just to make sure the pipeline works.